parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -321,6 +321,12 @@ ensureK8sControlPlane() {
     retrycmd_if_failure 120 5 25 $KUBECTL 2>/dev/null cluster-info || exit $ERR_K8S_RUNNING_TIMEOUT
 }
 
+ensureLabelExclusionForAzurePolicyAddon() {
+    if [[ "${azurePolicyAddonEnabled}" = True ]]; then
```

Comment:
Where are we deriving the `$azurePolicyAddonEnabled` env var?

Reply:
https://github.com/Azure/aks-engine/pull/2173/files#diff-de95d09bcc65a2843f3bad212bdac8e8R501

Reply:
Those are ARM variables, they won't (as far as I know) be exposed to CSE. That's why we create the `provisionScriptParametersCommon` and `provisionScriptParametersMaster` ARM variables, which we use to create the set of bash env vars to attach to the CSE execution context.

Reply:
thanks, updated